CMake/abslConfig.cmake.in

```diff
@@ -0,0 +1,7 @@
+## absl CMake configuration file
+
+set(ABSL_VERSION @PROJECT_VERSION@)
+
+@PACKAGE_INIT@
+
```

Comment:
It would be nice to have more meta data here, e.g.

```cmake
set_and_check(@PROJECT_NAME@_HOME "${PACKAGE_PREFIX_DIR}")
set_and_check(@PROJECT_NAME@_INCLUDEDIR "${PACKAGE_PREFIX_DIR}/@CMAKE_INSTALL_INCLUDEDIR@")
set_and_check(@PROJECT_NAME@_LIBDIR "${PACKAGE_PREFIX_DIR}/@CMAKE_INSTALL_LIBDIR@")

set(@PROJECT_NAME@_TARGETS absl::* absl::*_internal)
set(@PROJECT_NAME@_PUBLIC_TARGETS absl::*)
set(@PROJECT_NAME@_PRIVATE_TARGETS absl::*_internal)
```

The paths may be extracted from the target properties, but experience shows, it is nicer to have them at hand in a convenient variable (`${PACKAGE_PREFIX_DIR}` is populated by the code generated via `@PACKAGE_INIT@` at import time).

The entries in the proposed target lists could e.g. easily be collected in the `absl_cc_library` function.

Note: `@PROJECT_NAME@` is case-sensitive. You may wish to [convert to upper case](https://cmake.org/cmake/help/latest/command/string.html#toupper) characters.